Sources/Moya/Plugins/NetworkLoggerPlugin.swift

```diff
@@ -78,8 +80,9 @@ private extension NetworkLoggerPlugin {
             output += [format(loggerId, date: date, identifier: "HTTP Request Method", message: httpMethod)]
         }
 
-        if let body = request?.httpBody, let stringOutput = String(data: body, encoding: .utf8), isVerbose {
-            output += [format(loggerId, date: date, identifier: "Request Body", message: stringOutput)]
+        if let body = request?.httpBody, isVerbose {
```

Comment:
This line below is kinda unused if we do have our own formatter. Also in case of lack of message (`nil`), we still print request body (which we didn't before). How about this little change to `if let`:
```swift
if let body = request?.httpBody, let stringOutput = requestDataFormatter?(body) ?? String(data: body, encoding: .utf8), isVerbose {...}
```
Let me know what do you think.